org.hl7.fhir.validation/src/main/java/org/hl7/fhir/validation/cli/model/CliContext.java

```diff
@@ -328,6 +330,16 @@ public void setAllowDoubleQuotesInFHIRPath(boolean allowDoubleQuotesInFHIRPath)
     this.allowDoubleQuotesInFHIRPath = allowDoubleQuotesInFHIRPath;
   }
 
+  @JsonProperty("disableDefaultResourceFetcher")
+  public boolean isDisableDefaultResourceFetcher() {
+    return disableDefaultResourceFetcher;
+  }
+
+  @JsonProperty("disableDefaultResourceFetcher")
+  public void setDisableDefaultResourceFetcher(boolean disableDefaultResourceFetcher) {
```

Comment:
One detail here. This should be a fluent setter, and return CliContext so that CliContext settings can be chained (like `new CliContext().setDisableDefaultResourceFetcher(true).setCheckIPSCodes(false)`)

It should look like this one:

```java
@JsonProperty("bundleValidationRules")
  public CliContext setBundleValidationRules(List<BundleValidationRule> bundleValidationRules) {
    this.bundleValidationRules = bundleValidationRules;
    return this;
  }
```